Library/Formula/antlr.rb

```diff
@@ -13,4 +11,23 @@ def install
       java -classpath #{prefix}/antlr-#{version}-complete.jar:. org.antlr.v4.runtime.misc.TestRig "$@"
     EOS
   end
+
+  test do
+    path = testpath/"Expr.g4"
+    path.write <<-EOS.undent
+    grammar Expr;
+    prog:\t(expr NEWLINE)* ;
+    expr:\texpr ('*'|'/') expr
+        |\texpr ('+'|'-') expr
+        |\tINT
+        |\t'(' expr ')'
+        ;
+    NEWLINE :\t[\\r\\n]+ ;
+    INT     :\t[0-9]+ ;
+    EOS
+    ENV["CLASSPATH"] = ".:#{prefix}/antlr-#{version}-complete.jar:#{ENV["CLASSPATH"]}"
```

Comment:
Long-term we should probably drop the File.directory? check from this method entirely. For now what you have is fine.